docs/18_github_actions_ci_integration.md

```diff
@@ -0,0 +1,229 @@
+# CI Integrations GitHub Actions integration.
+
+> Summary of the proposed change
+
+Describe the integration mechanism of the CI Integrations component to the GitHub Actions to automatically push the build data.
+
+# References
+
+> Link to supporting documentation, GitHub tickets, etc.
+
+- [CI integrations](metrics/ci_integrations/docs/01_ci_integration_module_architecture.md)
+
+# Motivation
+
+> What problem is this project solving?
+
+This document describes the configuration process of the CI Integrations to automatically synchronize the GitHub Actions build data to the Metrics Web application.
+
+# Goals
+
+> Identify success metrics and measurable goals.
+
+This document aims the following goals: 
+
+- Explain the CI Integrations configuration process.
+- Explain the mechanism of automation build imports using the GitHub Actions and CI Integrations tool.
+
+# Non-Goals
+
+> Identify what's not in scope.
+
+This document does not describe the configuration of building or publishing jobs.
+
+# Design
+
+> Explain and diagram the technical design
+>
+> Identify risks and edge cases
+
+To be able to track the state of the applications under development, we should create `YAML` configuration files for each project that will be used by the CI Integrations tool to sync the build data with the Metrics Web application. Assume we have an `Awesome Project` and an `awesome_actions.yml` workflow file with `Awesome Project Actions` job used to run tests of the `Awesome Project`. If so, let's create a sample configuration file for this project. The configuration should consist of the following parts: 
```

Comment:
```suggestion
To be able to synchronize build data using the CI Integrations tool we should create `YAML` configuration files for each project that will be used by the CI Integrations tool to sync the build data with the Metrics Web application. Assume we have an `Awesome Project` and an `awesome_actions.yml` workflow file with `Awesome Project Actions` job used to run tests of the `Awesome Project`. If so, let's create a sample configuration file for this project. The configuration should consist of the following parts: 
```

maybe we can refer regular CI Integrations tool configuration document and focus on Github Actions nuances? Like storing some sensitive information in secrets and maybe just providing a sample configuration file?